contents/handbook/people/hiring-process.md

```diff
@@ -134,9 +134,20 @@ We will pay the candidate their 'normal day rate.' If they have done contracting
 
 This day will be _the same_ task each time for a given role, to be shared with the candidate at the start of the day. It should be _too much_ work for one person to complete in a day, in order to get a sense of the person's ability to prioritize. The tasks should be as close as possible to those that the candidate would be working on every day.
 
+In advance of the SuperDay, you will need to do some additional prep to ensure that the candidate has a great experience:
+
+* Send them an email in the first instance to schedule the SuperDay - you should do this as soon as possible, as candidates often will need to book a day off work. Use the Workable email template for this. If the task involves them doing 'real' work for PostHog, you should ask them to check that their current employment contract permits this - we try to create fake tasks for this reason. 
+* (One day before the SuperDay) Send the candidate a follow up email with details of the task, and ask them for their day rate and bank details. There are different templates for this email in Workable, depending on the role - this will probably need customising. 
+* (One day before the SuperDay) Create a private channel in Slack for the candidate, you and anyone else relevant - this will be where they can chat to us over the course of the day if they have any questions etc. Invite the candidate as a single channel guest. You may need to add the candidate to one of our systems depending on the role, e.g. Workable for a recruiter SuperDay, but on the whole this should be minimised.
+* (One day before the SuperDay) Invite the candidate to a kickoff meeting with you at the start of the day, and our daily standup at the end. You may also want to have a proper wrap up with them immediately after the team standup. 
+* (One day before the SuperDay) Send the candidate the task - aim to send this late on the day before so they don't have time to prepare (you can Schedule Send your email in Gmail for this). 
```

Comment:
I'd reframe slightly.

"late on the day before so that some candidates won't happen to have more time to prepare than others and the process remains fair"

Else it sounds a little "we're trying to trip people up"

Reply:
Agreed, now updated